proofs/goal.ml

```diff
@@ -67,11 +62,10 @@ module V82 = struct
 		Evd.evar_filter = Evd.Filter.identity;
 		Evd.evar_body = Evd.Evar_empty;
 		Evd.evar_source = (Loc.tag Evar_kinds.GoalEvar);
-		Evd.evar_candidates = None;
-		Evd.evar_extra = extra }
+                Evd.evar_candidates = None }
```

Comment:
indentation seems weird

Reply:
Right.

Reply:
That's because the new line doesn't use tabs, shall I untabify the rest?

Reply:
as you prefer